### PR TITLE
init

### PR DESCRIPTION
**Pull request by ekwanNTT**

### CODE REVIEW

File: adobe-common-application/pom.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        <dependency>
+            <groupId>ca.bc.gov.open</groupId>
+            <artifactId>adobe-common-ws-models</artifactId>
+            <version>1.0.0</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <finalName>jag-coa-application</finalName>
```
Comment (sylar-zeyangz) [inline adobe-common-application/pom.xml:196]: change needed

File: adobe-common-application/pom.xml [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.7</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/configuration/WSDLFilter*</exclude>
+                        <exclude>**/configuration/HttpServletRequestWrapper.class</exclude>
+                        <exclude>**/CoaApplication.class</exclude>
```
Comment (sylar-zeyangz) [inline adobe-common-application/pom.xml:249]: Change needed

File: adobe-common-application/src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        </filter>
+    </appender>
+
+    <appender name="SPLUNK" class="com.splunk.logging.HttpEventCollectorLogbackAppender">
+        <!--https://127.0.0.1:8088 for local-->
+        <url>${SPLUNK_HTTP_URL}</url>
+        <token>${SPLUNK_TOKEN}</token>
+        <index>${SPLUNK_INDEX}</index>
+        <source>coa-application</source>
+        <sourcetype>coa-logs</sourcetype>
```
Comment (sylar-zeyangz) [inline adobe-common-application/src/main/resources/logback.xml:22]: change needed

**Review by sylar-zeyangz — CHANGES_REQUESTED** [review]

File: adobe-common-application/pom.xml (hunk 1 above)
Comment (ekwanNTT) [reply adobe-common-application/pom.xml]: Fixed

File: adobe-common-application/pom.xml (hunk 2 above)
Comment (ekwanNTT) [reply adobe-common-application/pom.xml]: Fixed

File: adobe-common-application/src/main/resources/logback.xml (hunk 3 above)
Comment (ekwanNTT) [reply adobe-common-application/src/main/resources/logback.xml]: Fixed

**Review by sylar-zeyangz — COMMENTED** [review]
approved

**Pull request merged**